Assets/MRTK/Core/Utilities/Editor/ConfigurationStage.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    public enum ConfigurationStage
```

Comment:
missing docs

Reply:
though also, I wonder if there's value in this being public or if it could be internal with `InternalsVisibleTo` the Inspector assembly or something